cf-custom-resources/lib/unique-aliases.js

```diff
@@ -0,0 +1,141 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I wonder if we should rename the function to `unique-json-values` since it's pretty generic and not just applicable to aliases. The return key can be also `UniqueValues` 💭